src/export.js

```diff
@@ -47,8 +47,11 @@ Object.defineProperty( QUnit, "reset", {
 } );
 
 if ( defined.document ) {
-	if ( window.QUnit ) {
-		throw new Error( "QUnit has already been defined." );
+	if ( window.QUnit && window.QUnit.test ) {
+		throw new Error(
+			"QUnit has already been defined.\n" +
+			"Only a global config is allowed: QUnit = { config: { autostart: false, ...} }"
```

Comment:
I'm not sure this updated message is relevant. This error should only occur when `QUnit` has accidentally been loaded twice, shouldn't ever show up when someone is trying to do a custom config.